askup/tests/test_functional/test_general.py

```diff
@@ -27,6 +27,21 @@
 log = logging.getLogger(__name__)
 
 
+class GeneralTestCase(TestCase):
+    """
+    General test case class inherited by all the functional test cases below.
+    """
+
+    fixtures = ['groups', 'mockup_data']
+
+    def setUp(self):
+        """
+        Set up the default test assets.
+        """
+        settings.DEBUG = False
+        self.default_login()
```

Comment:
Do you need to do `default_login` in all cases?

Reply:
It was a noticeable over the half of the cases that this default admin login was used, last time I've checked it. It will take a time to review and refactor this moment. Do you want me to do it right now?

Reply:
All test passed so No ))
later.